### DISCUSSION TITLE
ET and BT swiched  Giesen W6A

### DISCUSSION THREAD

**Discussion by redadder22** [artisan-roaster-scope/artisan#1543] [Q&A]
Hello,

My BT and ET data is inverted: BT is showing ET data and ET is showing BT data.
Any idea why ?

Many thanks.

**Comment by redadder22** [comment]
<img width="969" alt="Screenshot 2024-04-26 at 14 17 32" src="https://github.com/artisan-roaster-scope/artisan/assets/29542458/d56ef5b0-6ef3-43f7-8fe4-6fce02305e8e">

**Comment by MAKOMO** [comment]
Maybe you have "ET <-> BT" ticked under menu Config >> Curves, second tab "Filter", section "Input Filter"? If not, tick it.

**Comment by redadder22** [comment]
That was it! thank you!
